custom_components/openevse/const.py

```diff
@@ -46,7 +46,7 @@
 DOMAIN = "openevse"
 COORDINATOR = "coordinator"
 FW_COORDINATOR = "fw_coordinator"
-VERSION = "1.0.0"
+VERSION = "2.1.26"
```

Comment:
```suggestion
VERSION = "1.0.0"
```

This change isn't needed as the github workflow updates the version on release.